common/persistence/pinotVisibilityTripleManager.go

```diff
@@ -344,6 +344,18 @@ func (v *pinotVisibilityTripleManager) logUserQueryParameters(userParam userPara
 		tag.VisibilityQuery(userParam.customQuery))
 }
 
+func (v *pinotVisibilityTripleManager) chooseManagerByOverride(ctx context.Context, domain string) VisibilityManager {
+	if override := ctx.Value(ContextKey); override == Primary {
+		panic("primary was chosen")
```

Comment:
avoid using panics and propagate errors to callers

Reply:
This is only for a testing purpose and will never be merged to the main